lib/i18n/hygiene/key_usage_checker.rb

```diff
@@ -7,8 +7,6 @@ def initialize(directories:, exclude_files: [], file_extensions: [])
         @directories = directories
         @exclude_files = exclude_files
         @file_extensions = file_extensions
-
-        raise "Must have git installed!" unless system("which git > /dev/null")
```

Comment:
Should we keep this guard, but update it to check for `grep`?

Reply:
Good idea. Will update!

Reply:
Done @ags!